src/GraphBLAS-sharp/Semiring.fs

```diff
@@ -0,0 +1,16 @@
+namespace GraphBLAS.FSharp
+
+type Semiring<'a> = {
```

Comment:
Read carefully these three definitions of semiring:
1. https://ru.wikipedia.org/wiki/%D0%9F%D0%BE%D0%BB%D1%83%D0%BA%D0%BE%D0%BB%D1%8C%D1%86%D0%BE
2. https://en.wikipedia.org/wiki/Semiring
3. https://people.eecs.berkeley.edu/~aydin/GraphBLAS_API_C_v13.pdf (Section 3.3 Semitings)

They are different. We should discuss, which one we should use.